python/tests/test_topology.py

```diff
@@ -49,6 +49,532 @@ def generate_segments(n, sequence_length=100, seed=None):
     return segs
 
 
+def kc_distance_tree(tree1, tree2, topo_v_age=0):
+    """
+    Returns the Kendall-Colijn distance between the specified
+    pair of trees. topology_vs_age determines weight of topology vs
+    branch lengths in calculating the distance.
```

Comment:
Maybe add a paper citation. This is the one I'm looking at: https://academic.oup.com/mbe/article/33/10/2735/2925548

Reply:
Good idea